symfony/framework-bundle/3.3/etc/routing.yaml

```diff
@@ -2,7 +2,7 @@
 #    path: /
 #    defaults: { _controller: 'App\Controller\DefaultController::index' }
 
-# Depends on sensio/framework-extra-bundle:^3.0 and doctrine/annotations
+# requires installing the 'sensio/framework-extra-bundle:^3.0' package
```

Comment:
The @Route, @Method, @Cache, etc. annotations are defined in the FwExtraBundle, not in the Doctrine thing.